tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -138,15 +145,24 @@ def parse_output(predictions, cells, num, box_size, anchors, processor, threshol
         if processed_box.confidence < threshold:
             continue
 
-        classes_prob = processed_box.probabilities
-        label = np.argmax(classes_prob)
+        if multi_label:
+            for class_id, probability in enumerate(processed_box.probabilities):
+                labels.append(class_id)
+                scores.append(probability * processed_box.confidence)
+                x_mins.append(processed_box.x - processed_box.w / 2.0)
+                y_mins.append(processed_box.y - processed_box.h / 2.0)
+                x_maxs.append(processed_box.x + processed_box.w / 2.0)
+                y_maxs.append(processed_box.y + processed_box.h / 2.0)
+        else:
+            classes_prob = processed_box.probabilities
+            label = np.argmax(classes_prob)
 
-        labels.append(label)
-        scores.append(processed_box.probabilities[label] * processed_box.confidence)
-        x_mins.append(processed_box.x - processed_box.w / 2.0)
-        y_mins.append(processed_box.y - processed_box.h / 2.0)
-        x_maxs.append(processed_box.x + processed_box.w / 2.0)
-        y_maxs.append(processed_box.y + processed_box.h / 2.0)
+            labels.append(label)
+            scores.append(processed_box.probabilities[label] * processed_box.confidence)
+            x_mins.append(processed_box.x - processed_box.w / 2.0)
+            y_mins.append(processed_box.y - processed_box.h / 2.0)
+            x_maxs.append(processed_box.x + processed_box.w / 2.0)
+            y_maxs.append(processed_box.y + processed_box.h / 2.0)
```

Comment:
possibly it is also should be refactored to use separated function/method for avoid extra branching